frontend/src/app/testQueue/TestQueue.tsx

```diff
@@ -97,12 +97,19 @@ interface QueueItemData {
 }
 
 const TestQueue: React.FC<Props> = ({ activeFacilityId }) => {
-  const { data, loading, error, refetch: refetchQueue } = useQuery(queueQuery, {
-    fetchPolicy: "no-cache",
-    variables: {
-      facilityId: activeFacilityId,
-    },
-    pollInterval,
+  const { data, loading, error, refetch, startPolling, stopPolling } = useQuery(
+    queueQuery,
+    {
+      fetchPolicy: "cache-and-network",
+      variables: {
+        facilityId: activeFacilityId,
+      },
+    }
+  );
+
+  useEffect(() => {
```

Comment:
Is the lack of a dependency array here intentional?

Reply:
Oh good catch. This _should_ work, but it will stop/start polling on every render which shouldn't be necessary. I just want an empty dependency array here instead, so that it only starts polling on first render and stops when the component is removed. I'll have to check that works but it should.